test/system/600-completion.bats

```diff
@@ -354,23 +354,29 @@ function _check_no_suggestions() {
         # check completion with relative path components
         # It is important the we will still use the image root and not escape to the host
         run_completion $cmd $IMAGE "../../"
-        assert "$output" =~ ".*^../../etc\$.*^../../home\$.*" "relative root directories suggested (cmd: podman $cmd ../../)"
+        assert "$output" =~ ".*^../../etc/\$.*^../../home/\$.*" "relative root directories suggested (cmd: podman $cmd ../../)"
     done
 
     random_name=$(random_string 30)
     random_file=$(random_string 30)
-    run_podman run --name $random_name $IMAGE touch /tmp/$random_file
+    run_podman run --name $random_name $IMAGE sh -c "touch /tmp/$random_file && touch /tmp/${random_file}2 && mkdir /emptydir"
```

Comment:
Simplest (safest, maybe) suggestion: change the first to `${random_file}1` (and following checks accordingly). Some locales sort `foo2` before `foo`, but there is no locale (that I know of) that will sort `foo2` before `foo1`